app/views/admin/users/index.html.erb

```diff
@@ -1,13 +1,13 @@
 <ul>
-  <% @users.each do |user| %>
+  <% @user.each do |current_user| %>
```

Comment:
I do not think we should call the block variable current_user, it's confusing since that's our helper method too.

I would change this to: `@users.each do |user|`

Reply:
Made this changed not sure why I switched it in the first place